scripts/livedebug.py

```diff
@@ -95,6 +100,12 @@ def livedebug():
     # 5. load sql file in psql pane if a file was provided
     time.sleep(1.3)
     if args.file:
+        # as our sql imports are done vie \ir sq/file.sql
+        # we should be in /tests directory so the imports
+        # will resolve the file paths correctly
+        file_dir = '/'.join(args.file.split('/')[:-2])
+        python_pane.send_keys(f"\cd {file_dir}")
```

Comment:
`split_window` above has a `start_directory` [option](https://libtmux.git-pull.com/reference/windows.html#libtmux.Window.split_window). Probably cleaner to use that instead of this